shared/iam_audit.py

```diff
@@ -33,7 +34,7 @@ def action_matches(action_from_policy, actions_to_match_against):
 
 
 def policy_action_count(policy_doc, location):
-    # Counts how many unrestricted actions a policy grants
+    """Counts how many unrestricted actions a policy grants"""
```

Comment:
I didn't look at all the code, I agree docstring format is more pythonic, but the other code that I saw doesn't use it so perhaps consistency is better 🤷🏼

Reply:
Hello @KevinHock ! I checked some of the code and since that comment was documenting the functionality of the whole function, I decided to change it.
Take a look at -> https://github.com/duo-labs/cloudmapper/pull/852/files
In some cases, docstring is used, but not in others, hence I assumed that consistency is not well defined,regarding documentation.
Since docstring is better to use it there, it's a small change which improves the code :)